guide/english/certifications/responsive-web-design/basic-css/improve-compatibility-with-browser-fallbacks/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Improve Compatibility with Browser Fallbacks
+---
+## Improve Compatibility with Browser Fallbacks
+
+To avoid compatibility issues with browsers, it is best to specify properties in case.
+
+Here instead of this code:
+```css
+.red-box {
+background: var(--red-color);
+height: 200px;
+width:200px;
+}
+```
+
+
+It is better to add a solution in case the browser is incompatible with this property:
```

Comment:
It would make much more sense to use the word fallback instead of solution here. Or fallback solution.